app/docs/enterprise/0.31-x/installation/ubuntu.md

```diff
@@ -0,0 +1,78 @@
+---
+title: How to install Kong Enterprise and PostgreSQL onto Ubuntu
+---
+
+## Install Kong Enterprise
+
+Download the EE binary for Ubuntu (or Debian) from Bintray
+SFTP the file to your Ubuntu server
+
+```bash
+$ sudo apt-get update
+$ sudo apt-get install openssl libpcre3 procps perl
+$ sudo dpkg -i kong-enterprise-edition-0.31-1.zesty.all.deb
+```
+NOTE: EE file may differ in last step above
+
+## Install PostgreSQL
+```bash
+$ sudo apt-get install postgresql
+$ sudo -i -u postgres (puts you into new shell)
+```
+
+Create `kong` user
+
+```bash
+$ psql
+> CREATE USER kong; CREATE DATABASE kong OWNER kong; ALTER USER kong WITH password 'kong'; 
+> \q
+$ exit
+```
+
+## Finish Kong Enterprise setup steps
+
+### add contents of your license file
+```bash
+$ sudo vi /etc/kong/license.json
+```
+
+### Copy kong.conf.default to create kong.conf
+```bash
+$ sudo cp /etc/kong/kong.conf.default /etc/kong/kong.conf
+```
+### Uncomment and add 'kong' to pg_password line
+```bash
+$ sudo vi /etc/kong/kong.conf
+```
+
+### Run migrations and start kong
+```bash
+$ kong migrations up
+$ sudo /usr/local/bin/kong start
+```
+
+### Setup HTTPie to make commands easier
```

Comment:
Not required, just mention that for this you used `httpie` but user can use any http client like curl